src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "altinn-app-frontend",
-  "version": "3.1.6",
+  "version": "3.1.7",
```

Comment:
Maybe we should bump the minor version? Since this actually is new functionality.
I guess we generally should be a little better at using semver 👀

Reply:
Yup!